packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -714,8 +713,8 @@ const runOptimize = async () => {
 	};
 
 	// InferredParameters is to link a calibration run to this optimize call.
-	if (inferredParameters.value) {
-		optimizePayload.extra.inferredParameters = inferredParameters.value[0];
+	if (modelConfiguration.value) {
```

Comment:
We already check for modelConfiguration at the start of this function
We should be able to just add this to the payload when we define it a few lines above if this is the case

What do we send for a non-calibrated model? Has this been tested?
pyciemss-service expects None 
https://github.com/DARPA-ASKEM/pyciemss-service/blob/main/service/models/converters.py#L10-L24